masonry/src/render_root.rs

```diff
@@ -74,6 +74,7 @@ pub(crate) struct RenderRootState {
     pub(crate) focused_widget: Option<WidgetId>,
     pub(crate) focused_path: Vec<WidgetId>,
     pub(crate) next_focused_widget: Option<WidgetId>,
+    pub(crate) most_recently_clicked_widget: Option<WidgetId>,
```

Comment:
It might be worth having just a sprinkling of documentation/explanation here (the same could be said about most of these fields, but this one is a bit non-obvious)